src/Recruit.Vacancies.Client/Domain/Entities/VacancyExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Esfa.Recruit.Storage.Client.Domain.Enum;
+
+namespace Esfa.Recruit.Storage.Client.Domain.Entities
+{ 
+    public static class VacancyExtensions
+    {
+        public static bool IsSubmittable(this Vacancy vacancy)
```

Comment:
I think having this as a read-only property on the model would be fine e.g. `public bool IsSubmittale => Status == VacancyStatus.Draft;`

Its not complex logic or behavior you are introducing so I think it is fine to keep in Entity.Also you are only inspecting the state it already knows about, not the state of another object.

Reply:
Sounds like a plan. Will change